icontrol/session.py

```diff
@@ -423,7 +423,7 @@ def __init__(self, username, password, **kwargs):
             else:
                 if token_auth is True:
                     self.session.auth = iControlRESTTokenAuth(
-                        username, password, verify=verify
+                        username, password, verify=verify, timeout=timeout
```

Comment:
Do we also need to input timeout parameter for other two cases in line 420 and 430?